spec/services/recaptcha_service_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RecaptchaService do
+  let(:failed_attempts) { 0 }
+  let(:user) { create(:user, failed_attempts: failed_attempts) }
+
+  subject { described_class.new(user) }
+
+  describe '#recaptcha_configured?' do
+    let(:site_key) { '11111' }
+    let(:secret_key) { '22222' }
+    let(:configuration) {
+      instance_double(
+        Recaptcha::Configuration,
+        site_key: site_key,
+        secret_key: secret_key,
+      )
+    }
+
+    before do
+      allow(Recaptcha).to receive(:configuration).and_return(configuration)
+    end
+
+    context 'site key and secrete key are present' do
```

Comment:
```suggestion
    context 'when site key and secrete key are present' do
```

Also lines 33, 41, 51, 61